.github/run_browserstack_acceptance.sh

```diff
@@ -5,5 +5,6 @@ GITHUB_BRANCH=${GITHUB_REF#refs/heads/}
 export BROWSERSTACK_BUILD_ID="${GITHUB_BRANCH} - ${GITHUB_RUN_ID}"
 COMMIT_MSG_TITLE=$(git log -n 1 --pretty=format:%s)
 export BROWSERSTACK_TEST_RUN_NAME=$COMMIT_MSG_TITLE
+export IS_DEVELOPMENT_PREVIEW="false"
```

Comment:
I don't think setting the environment variable here does anything. At this point `npm run build-test-site` is already done right?

Reply:
oh, you're right. I would need to set it in the github actions for it to take effect in the build. Honestly though, prod mode is the default, so I don't really think it's necessary to set it